package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "stellar-base",
-  "version": "0.11.0",
+  "version": "0.12.0",
   "description": "Low level stellar support library",
   "main": "lib/index.js",
```

Comment:
Do you want to update `js-xdr` to the recently release version?

Reply:
Yup, just did!